lms/djangoapps/certificates/tests/test_signals.py

```diff
@@ -2,12 +2,15 @@
 Unit tests for enabling self-generated certificates for self-paced courses
 and disabling for instructor-paced courses.
 """
+import mock
+
 from certificates import api as certs_api
-from certificates.models import CertificateGenerationConfiguration
+from certificates.models import CertificateGenerationConfiguration, CertificateWhitelist
 from certificates.signals import _listen_for_course_pacing_changed
 from openedx.core.djangoapps.self_paced.models import SelfPacedConfiguration
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 from xmodule.modulestore.tests.factories import CourseFactory
+from student.tests.factories import UserFactory
```

Comment:
This line should move up to be in alphabetical order....how do we make isort do that automagically?